src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -260,7 +388,7 @@ public function list(array $filters = []): array
      * 
      * @return string
      */
-    public function run(string $image, string $name, array $command, string $entrypoint = '', string $workdir = '/', array $volumes = [], array $vars = [], string $mountFolder = '', array $labels = []): string
+    public function run(string $image, string $name, array $command = [], string $entrypoint = '', string $workdir = '', array $volumes = [], array $vars = [], string $mountFolder = '', array $labels = [], string $hostname = ''): string
```

Comment:
```suggestion
    public function run(
        string $image, 
        string $name, 
        array $command = [], 
        string $entrypoint = '', 
        string $workdir = '', 
        array $volumes = [], 
        array $vars = [], 
        string $mountFolder = '', 
        array $labels = [], 
        string $hostname = ''
    ): string {
```

When you have a long argument list, it's better to split it into multiple lines.
![Uploading Screenshot 2021-12-12 at 1.41.45 PM.png…]()